<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookstore Management App</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Logged in as</string>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>0</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Log out</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBar">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="modeChangeBtn">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode: Manage</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dataTable">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>581</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>50</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>150</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stockBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Out of stock/In stock</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>300</y>
      <width>91</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="syncBtn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>343</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sync</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
